lib/spider/scripts/in_context_learning.py

```diff
@@ -32,21 +34,27 @@
 db_dir = DATASET_PATH / "database"
 table = DATASET_PATH / "tables.json"
 
+nltk.download("punkt")
+
```

Comment:
I explicitly removed this since we don't use `nltk`, if there is no reason to add it back you can remove it!